qa/rbd/krbd_default_map.t

```diff
@@ -0,0 +1,37 @@
+Test at global level
+  $ OPTIONS="crc"
+  $ EFFECTIVE="${OPTIONS}"
+  $ ceph config set global rbd_default_map_options $OPTIONS
+  $ ceph osd pool create rbda
+  pool 'rbda' created
+  $ rbd pool init rbda
+  $ rbd create image1 --size 1000 --pool rbda
+  $ DEV=$(rbd map rbda/image1)
+  $ DEV_ID1=$(rbd device list | grep rbda | grep image1 | grep \- | awk '{print $1;}')
+  $ grep -q ${EFFECTIVE} /sys/bus/rbd/devices/$DEV_ID1/config_info
+  $ rbd unmap rbda/image1
+Test at pool level
```

Comment:
Add an empty line before and after these lines to make it easier to navigate between test cases.

Reply:
This is done.